context-free.md

```diff
@@ -2,21 +2,27 @@
 
 ##  Who is this programming language for?
 
+Context Free is for anyone artistically inclined who either has programming experience or is willing to learn some basic programming concepts.
```

Comment:
How is this different from using a language with a graphics library? What user would use ContextFree over those? Are there particular types of images that are more suited to be created using ContextFree?